source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -330,7 +330,166 @@ export class ContainerStyleSet {
     }
 }
 
+export class Version {
+    private _versionString: string;
+    private _major: number;
+    private _minor: number;
```

Comment:
shouldn't we be supporting semantic versioning? there should be a `build` and `revision` property

Reply:
There is an active discussion on the side about this. I think we should keep it simple, but I will update the code according to the conclusion of the conversation.

Reply:
Per discussion we are going with a simple model for 1.2.